### DISCUSSION TITLE
在相同的起始节点和终止之间创建多条边,设置了allowMulti为true 但是不生效

### DISCUSSION THREAD

**Discussion by h5-Old-driver** [antvis/X6#3183] [Q&A]
`connecting: {
                  allowBlank: false,
                  allowMulti: true,
                  allowLoop: true,
                  allowNode: false,
                  allowEdge: false,
                  highlight: true,
                  connector: 'rounded',
                  snap: true,},`

allowMulti 已经设置为true了   在相同的起始节点和终止之间依然不能创建多条边

**Comment by ChenSheng233** [comment]
我也有这种场景， new Graph 的 validateEdge、validateConnection、createEdge
![image](https://user-images.githubusercontent.com/39158941/221177517-4ef1d202-1bf8-4925-b4c3-1157e2286f2c.png)
